<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Chris York</author>
 <class>QGISpellDockWidgetBase</class>
 <widget class="QDockWidget" name="QGISpellDockWidgetBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>462</width>
    <height>434</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QGISpell</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QFrame" name="fra_main">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QTabWidget" name="Tabs">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>461</width>
         <height>401</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Candara</family>
        </font>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab_layers">
        <attribute name="title">
         <string>Layer selection</string>
        </attribute>
        <widget class="QgsMapLayerComboBox" name="MapLayer">
         <property name="geometry">
          <rect>
           <x>96</x>
           <y>12</y>
           <width>261</width>
           <height>27</height>
          </rect>
         </property>
        </widget>
        <widget class="QgsFieldComboBox" name="FieldOrderBy">
         <property name="geometry">
          <rect>
           <x>96</x>
           <y>62</y>
           <width>261</width>
           <height>27</height>
          </rect>
         </property>
        </widget>
        <widget class="QRadioButton" name="rad_action_pan">
         <property name="geometry">
          <rect>
           <x>96</x>
           <y>142</y>
           <width>91</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Pan</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QRadioButton" name="rad_action_zoom">
         <property name="geometry">
          <rect>
           <x>266</x>
           <y>142</y>
           <width>91</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Zoom</string>
         </property>
        </widget>
        <widget class="QLabel" name="labelLayer">
         <property name="geometry">
          <rect>
           <x>6</x>
           <y>12</y>
           <width>81</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Select Layer</string>
         </property>
        </widget>
        <widget class="QLabel" name="labelOrderBy">
         <property name="geometry">
          <rect>
           <x>6</x>
           <y>62</y>
           <width>81</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Order by</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="chk_use_sel">
         <property name="geometry">
          <rect>
           <x>96</x>
           <y>112</y>
           <width>131</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Use selected only</string>
         </property>
        </widget>
        <widget class="QPushButton" name="layerRefresh">
         <property name="geometry">
          <rect>
           <x>366</x>
           <y>12</y>
           <width>61</width>
           <height>28</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Candara</family>
          </font>
         </property>
         <property name="text">
          <string>Refresh</string>
         </property>
        </widget>
        <widget class="QLabel" name="Canvas">
         <property name="geometry">
          <rect>
           <x>6</x>
           <y>142</y>
           <width>55</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Canvas</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_spelling">
        <attribute name="title">
         <string>Spell checking</string>
        </attribute>
        <widget class="QScrollArea" name="scrollArea">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>50</y>
           <width>451</width>
           <height>241</height>
          </rect>
         </property>
         <property name="sizeAdjustPolicy">
          <enum>QAbstractScrollArea::AdjustToContents</enum>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>449</width>
            <height>239</height>
           </rect>
          </property>
          <widget class="QWidget" name="formLayoutWidget">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>451</width>
             <height>231</height>
            </rect>
           </property>
           <layout class="QFormLayout" name="formLayout"/>
          </widget>
         </widget>
        </widget>
        <widget class="QPushButton" name="btn_last">
         <property name="geometry">
          <rect>
           <x>280</x>
           <y>23</y>
           <width>31</width>
           <height>24</height>
          </rect>
         </property>
         <property name="text">
          <string>&gt;|</string>
         </property>
        </widget>
        <widget class="QPushButton" name="btn_next">
         <property name="geometry">
          <rect>
           <x>240</x>
           <y>23</y>
           <width>31</width>
           <height>24</height>
          </rect>
         </property>
         <property name="text">
          <string>&gt;</string>
         </property>
        </widget>
        <widget class="QPushButton" name="btn_prev">
         <property name="geometry">
          <rect>
           <x>200</x>
           <y>23</y>
           <width>31</width>
           <height>24</height>
          </rect>
         </property>
         <property name="text">
          <string>&lt;</string>
         </property>
        </widget>
        <widget class="QPushButton" name="btn_first">
         <property name="geometry">
          <rect>
           <x>160</x>
           <y>23</y>
           <width>31</width>
           <height>24</height>
          </rect>
         </property>
         <property name="text">
          <string>|&lt;</string>
         </property>
        </widget>
        <widget class="QLabel" name="CountFeatures">
         <property name="geometry">
          <rect>
           <x>330</x>
           <y>23</y>
           <width>131</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>- of -- features</string>
         </property>
        </widget>
        <widget class="QPushButton" name="save_record">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>23</y>
           <width>61</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Accept</string>
         </property>
        </widget>
        <widget class="QPushButton" name="cancel_record">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>23</y>
           <width>61</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
        <widget class="QLabel" name="layer_label">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>451</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Layer:</string>
         </property>
         <property name="indent">
          <number>5</number>
         </property>
        </widget>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsFieldComboBox</class>
   <extends>QComboBox</extends>
   <header>qgsfieldcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>QgsMapLayerComboBox</class>
   <extends>QComboBox</extends>
   <header>qgsmaplayercombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
